components/script/dom/readablestreamdefaultreader.rs

```diff
@@ -336,6 +336,43 @@ impl ReadableStreamDefaultReader {
             .borrow()
             .append_native_handler(&handler, comp, can_gc);
     }
+
+    /// <https://streams.spec.whatwg.org/#readablestreamdefaultreader-read-all-bytes>
+    pub(crate) fn read_all_bytes(
+        &self,
+        promise: Rc<Promise>,
+        success_steps: ReadAllBytesSuccessSteps,
+        failure_steps: ReadAllBytesFailureSteps,
+        comp: InRealm,
+        can_gc: CanGc,
+    ) {
+        let global = self.global();
+
+        let read_promise = self.Read(can_gc);
```

Comment:
This should be documented with the sentences from the two steps of https://streams.spec.whatwg.org/#readablestreamdefaultreader-read-all-bytes

and with a further comment explaining that instead of using a dedicated read request, we use the handles attached to the promise returned by `Read` to simulate the read request logic specified in the spec.